app/components/footer_component.html.erb

```diff
@@ -41,7 +105,9 @@
         <div class="site-footer-bottom__links-container__license">All content is available under the <a href="https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/">Open Government Licence v3.0,</a> except where otherwise stated.</div>
       </div>
       <div class="site-footer-bottom__logo-container">
-        <%= image_pack_tag 'static/images/dfelogo-white.svg', alt: "Department for Education logo", size: "124x73" %>
+        <%= link_to root_path do %>
```

Comment:
@spencerldixon Link needs to go to the DfE home page (not ours!) which is `https://www.gov.uk/government/organisations/department-for-education`